test/ui-test/testSuites/suite_messaging/tst_chatsNavigation/test.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Status Desktop Chat Navigation
+
+    Scenario: The user joins a room and marks it as read
+        When the user joins chat room "test"
+        And the user marks the channel "test" as read
+        # TODO find a way to validate that it worked
+
+    @merge
```

Comment:
Remove this tag since you've already added the `onFeatureStart` steps, it can be enabled in the ci bc the time-consuming steps have been reduced.